### ISSUE TITLE
The user can download their PDF draft comment from this page

### ISSUE THREAD

**Issue by Journerdism** [eregs/notice-and-comment#205]
Write mode
[NEEDED] The user can download their PDF draft comment from this page.

**Comment by jehlers** [comment]
Maybe this should be added under the review and submit button? Or under the text area for the comment.

**Comment by cmc333333** [comment]
I don't follow, @jehlers. Is the idea that the user can see the _full_ PDF before the "review" page?

**Comment by jehlers** [comment]
The idea is that someone may want to change computers or save a copy of the full comment to share with a co-worker. They will not have gotten to the review page yet, so might not know they can download the draft doc. That was the thought process for pulling the same button for building and downloading the pdf onto the write mode page in addition to the review page.

**Comment by vrajmohan** [comment]
This issue either duplicates or is related to eregs/notice-and-comment#124.

**Comment by cmc333333** [comment]
If I understand @jehlers correctly, this would _leverage_ #124, but be about placing the link on each "write" page (as opposed to just the "review" page)

**Comment by jehlers** [comment]
Yes. It would download the draft that you are working on @vrajmohan, but this issue is talking about having the link to download that doc on both the write page and review page.
